content/docs/tasks/security/auth-sds/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Identity provision through SDS
+description: Shows how to enable SDS(secret discovery service) for istio identity provision.
+weight: 70
+keywords: [security,auth-sds]
+---
+
+This task shows how to enable [SDS(secret discovery service)](https://www.envoyproxy.io/docs/envoy/latest/configuration/secret#config-secret-discovery-service) for istio identity provision.
+
+Today the keys and certificates of Istio workloads are generated by Citadel and distributed to sidecars
```

Comment:
This is the 1.1 doc set, so change this section to be in the past. Something like:

Prior to Istio 1.1, ...